idaes/models_extra/column_models/util.py

```diff
@@ -286,13 +286,7 @@ def rule_flow(model, t, p, i):
 
                 model.add_component("e_" + local_name + port.local_name, expr)
                 port.add(expr, k)
-            else:
```

Comment:
Just to check, are you sure this is no longer needed?

Reply:
@andrewlee94 maybe you could give me more insight but my understanfing is that `if "phase" not in local_name:` and `elif "phase" in local_name:` are the only options available as it is in the local_name or is not but I may be wrong

Reply:
Those are the options, however in that case the `elif` really should just be an `else`. I was mostly asking as it is difficult to follow the logic in a big statement like this so I wanted to be sure there was no chance of an overrun error.